crates/ra_env/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+edition = "2018"
+name = "ra_env"
+version = "0.1.0"
+authors = ["rust-analyzer developers"]
+
+[dependencies]
+anyhow = "1.0.26"
+dirs = "2.0"
```

Comment:
I feel it's better to keep this as a module in `rust-analyzer` crate rather than a separate crate.

Reply:
Ah, we use it in more than one crate.... let me thing about where is it best to put this then...

Reply:
Maybe we should actually pull additional functionality into this new crate.  For instance, sysroot.rs has a function `run_command_in_cargo_dir`:
https://github.com/rust-analyzer/rust-analyzer/blob/9cb55966fe0fee791072f275ac55b90b8ee13e32/crates/ra_project_model/src/sysroot.rs#L91
which maybe the other callsites could benefit from, I'm not sure.